src/api/java/com/minecolonies/api/entity/ai/citizen/guards/GuardItems.java

```diff
@@ -0,0 +1,158 @@
+package com.minecolonies.api.entity.ai.citizen.guards;
+
+import com.minecolonies.api.util.ItemStackUtils;
+import com.minecolonies.api.util.constant.IToolType;
+import net.minecraft.inventory.EntityEquipmentSlot;
+import net.minecraft.inventory.ItemStackHelper;
+import net.minecraft.item.ItemArmor;
+import net.minecraft.item.ItemShield;
+import net.minecraft.item.ItemStack;
+
+/**
+ * Class to hold information about required item for the guard.
+ */
+public class GuardItems
```

Comment:
Since this class has a lot of configurable options, consider creating a Builder class instead of using the constructor.